src/nbl/video/COpenGLFeatureMap.h

```diff
@@ -1156,14 +1156,8 @@ class COpenGLFeatureMap
 	//!
 	int32_t reqTBOAlignment;
```

Comment:
`minTexelBufferOffsetAlignment`

Reply:
for the UBO and SSBO its `minUniformBufferOffsetAlignment` and `minStorageBufferOffsetAlignment` respectively